suites/suites/suites.yaml

```diff
@@ -389,8 +391,10 @@ handler_properties:
   aws_ec2:
     ubuntu_trusty_image_id: '{{aws_ec2_eu_central_1_ami_ubuntu_trusty_image_id}}'
     centos_7_image_id: '{{aws_ec2_eu_central_1_ami_centos_7_image_id}}'
+    aws_ec2_eu_west_1_ami_rhel_7_image_id: '{{aws_ec2_eu_west_1_ami_rhel_7_image_id}}'
```

Comment:
Please name it rhel_7_image_id.

Once we'll run the tests on different regions we will have a different set of properties per region and the image_id property name should be exactly the same so that we won't be limited to running a certain test on a certain region.